.run/1. Platform Server - ServiceRunner.run.xml

```diff
@@ -8,7 +8,7 @@
       <env name="SECRET.DATA.USERNAME" value="admin" />
       <env name="secret.sep10.jwt_secret" value="jwt_secret" />
       <env name="secret.sep10.signing_seed" value="SAKXNWVTRVR4SJSHZUDB2CLJXEQHRT62MYQWA2HBB7YBOTCFJJJ55BZF" />
-      <env name="STELLAR_ANCHOR_CONFIG" value="file:///$PROJECT_DIR$/integration-tests/src/test/resources/integration-test.anchor-config.yaml" />
+      <env name="STELLAR_ANCHOR_CONFIG" value="file:///$PROJECT_DIR$/platform/src/main/resources/example.anchor-config.yaml" />
```

Comment:
Please create a new configuration that runs the `example.anchor-config.yaml`. 

We may probably rename the file to `Platform Server (Integration Test Configuration)` and `Platform Server (Example Configuration)